packages/scripts/package.json

```diff
@@ -47,7 +47,7 @@
 		"jest-puppeteer": "^4.3.0",
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^3.6.0",
-		"puppeteer": "1.19.0",
+		"puppeteer": "^1.19.0",
```

Comment:
@gziolo Just wanting to confirm that adding the `^` caret here is deliberate?

Reply:
> In addition, it enables ranged for puppeteer dependency so we could update it more often with ease.

Yes, I also included it in the CHANGELOG file. In general, while working on upgrade we started at 1.15 and didn't see any issues up to 1.19, so there is no need to be worried about it.

Reply:
Why do we keep these if there is a lock file? Why to get rid of all `^` entirely?

Reply:
it's not for us, it's for those who consume this package on npm. They shouldn't be forced to use the same version we do.

Reply:
Thanks, all makes sense, merge when green 💯